lib/tabs/__tests__/KTabsList.spec.js

```diff
@@ -28,20 +37,29 @@ const TABS_WITH_ROUTES = [
 ];
 
 describe(`KTabsList`, () => {
-  beforeAll(() => {
-    jest.spyOn(console, 'error').mockImplementation(() => {});
-  });
+  let errorListener;
+
+  function makeWrapper({ propsData = {} } = {}) {
+    return mount(KTabsList, {
+      propsData,
+      localVue,
+      router,
+      listeners: {
+        error: errorListener,
+      },
+    });
+  }
 
-  afterAll(() => {
-    console.error.mockRestore();
+  beforeEach(() => {
+    errorListener = jest.fn();
   });
 
   afterEach(() => {
-    console.error.mockClear();
+    errorListener = null;
```

Comment:
(Just curious) Is there any benefit to preferring this pattern to defining a single jest.fn() and doing errorListener.mockClear after each test?

Reply:
Oh, that's very interesting. Thanks!